Contents/mods/Hydrocraft/media/scripts/Seeds.txt

```diff
@@ -1501,7 +1501,7 @@ item HCBeetsugarseedpacket
 	{
         OnTest:HCNearHerbatable,
         keep HCSieve/HCFloursieve,
-		Popcorn/Cereal/HCHerbs,
+		Popcorn/Cereal/HCHerbs/HCDirt,
```

Comment:
are you intending to add a recipe to the HCFindSeeds function to find something from dirt? would give nothing atm with this